src/Metadata/Extractor/YamlExtractor.php

```diff
@@ -32,7 +32,7 @@
     protected function extractPath(string $path)
     {
         try {
-            $resourcesYaml = Yaml::parse(file_get_contents($path));
+            $resourcesYaml = Yaml::parse(file_get_contents($path), Yaml::PARSE_KEYS_AS_STRINGS);
```

Comment:
Yes it's correct, but this constant was introduced in `symfony/yaml` 3.3. So tests will fail with the `--prefer-lowest` argument!